packages/swagger-ui/README.md

```diff
@@ -0,0 +1,81 @@
+# Swagger UI Middleware and Component for Hono
+
+This library, `@hono/swagger-ui`, provides a middleware and a component for integrating Swagger UI with Hono applications. Swagger UI is an interactive documentation interface for RESTful APIs, making it easier to understand and test API endpoints.
+
+## Installation
+
+```bash
+npm install @hono/swagger-ui
+# or
+yarn add @hono/swagger-ui
+```
+
+## Usage
+
+### Middleware Usage
+
+You can use the `swaggerUI` middleware to serve Swagger UI on a specific route in your Hono application. Here's how you can do it:
+
+```ts
+import { Hono } from 'hono'
+import { swaggerUI } from '@hono/swagger-ui'
+
+const app = new Hono()
+
+// Use the middleware to serve Swagger UI at /ui
+app.get('/ui', swaggerUI({ url: '/doc' }))
+
+export default app
+```
+
+### Component Usage
+
+If you are using `hono/utils/html`, you can use the `SwaggerUI` component to render Swagger UI within your custom HTML structure. Here's an example:
```

Comment:
`hono/utils/html` should be `hono/html`.

Reply:
Oh, sorry. I omitted a correction.